src/Gateway/PayplugGateway.php

```diff
@@ -554,7 +554,6 @@ public function integrated_payments_scripts(){
 			"cvv" =>  __('payplug_integrated_payment_cvv', 'payplug'),
 			"one_click" =>  __('payplug_integrated_payment_oneClick', 'payplug'),
 			'ajax_url' => \WC_AJAX::get_endpoint('payplug_create_order'),
-			'check_payment_url' => \WC_AJAX::get_endpoint('payplug_check_payment'),
```

Comment:
you can delete from the PHP side the action `wc_ajax_payplug_check_payment` as well as the method `check_payment` because this was his only use